src/main.rs

```diff
@@ -111,6 +115,7 @@ fn run() -> Result<()> {
 	options.optopt("p", "port", "set polaris to run on a custom port", "PORT");
 	options.optopt("d", "database", "set the path to index database", "FILE");
 	options.optopt("w", "web", "set the path to web client files", "DIRECTORY");
+	options.optopt("l", "log", "set the log level", "LEVEL");
```

Comment:
Can you mention that the log levels go from 0 (off) to 3 (debug) in the help message?